### PR TITLE
feat: Allow shortening symbols from multi-use statements (only classes for now)

### PR DESCRIPTION
**Pull request by Wirone**
Support multi-use import statements where FQCN shortening is currently possible.

### CODE REVIEW

File: tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php [hunk 1]
```diff
@@ -1090,6 +1090,54 @@ public function getValue()
             null,
             ['import_symbols' => true],
         ];
+
+        // TODO: Ensure shortening for imported functions and constants
```
Comment (julienfalque) [inline tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php:1094]: Is shortening functions and constants not currently supported at all by the rule?

File: tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php [hunk 2]
```diff
@@ -1090,6 +1090,54 @@ public function getValue()
...
+                use function Bar\func1, Bar\func2;
+                use const Bar\CONST1, Bar\CONST2;
+
+                \Bar\func1(new \Bar\Service1(\Bar\CONST1));
+                \Bar\func2(new \Bar\Service2(\Bar\CONST2));
+                EOD,
+        ];
+
+        // TODO: Ensure shortening for imported functions and constants
+        yield 'Shorten symbol from grouped multi-use statement' => [
```
Comment (julienfalque) [inline tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php:1119]: Can you add a similar case with newlines between grouped symbols?

File: tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php (hunk 1 above)
Comment (Wirone) [reply tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php:1094]: Yes, the rule currently can shorten only class-like symbols. See #7460.

File: tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php (hunk 2 above)
Comment (Wirone) [reply tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php:1149]: Did you mean something like [this](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7816/commits/0dab597e17190476bdb6f75e227effc7e1370baa)?

Comment (julienfalque) [reply tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php:1149]: Yes 😄

**Review by julienfalque — APPROVED** [review]

**Pull request merged**